src/CommandBasedBackendRegisterAccessor.cpp

```diff
@@ -67,11 +67,18 @@ namespace ChimeraTK {
     }
 
     if(_registerInfo.internalType != CommandBasedBackendRegisterInfo::InternalType::VOID) {
+      if(valueRegex.empty()) {
```

Comment:
Well, this Debug output will go away anyway. But keep it in here so we have a debugging commit in out history.